assisted-installer.yaml

```diff
@@ -1,24 +1,24 @@
 openshift/assisted-installer:
   revision: c81363a7fada8d23cfbac86ecc0f97835f1fa043
   images:
-  - assisted-installer-controller
-  - assisted-installer
+  - quay.io/ocpmetal/assisted-installer-controller
+  - quay.io/ocpmetal/assisted-installer
 openshift/assisted-service:
```

Comment:
This should be edge-infrastructure..

Reply:
indeed ``assisted-installer`` can be changed, ``assisted-installer-controller`` not yet
once we'll trigger image-mirroring we can change it also

Reply:
fixed